docs/docs/building-apps/02-app-mempool.md

```diff
@@ -35,29 +37,83 @@ Process proposal handles the validation of what is in a block, meaning that afte
 Here is the implementation of the default implementation:
 
 ```go reference
-https://github.com/cosmos/cosmos-sdk/blob/bcff22a3767b9c5dd7d1d562aece90cf72e05e85/baseapp/baseapp.go#L906-L930
+https://github.com/cosmos/cosmos-sdk/blob/v0.47.0-alpha2/baseapp/baseapp.go#L911-L935
 ```
 
-Like `PrepareProposal` this implementation is the default and can be modified by the application developer. 
+Like `PrepareProposal` this implementation is the default and can be modified by the application developer in [`app.go`](./01-app-go-v2.md):
 
-```go reference
-https://github.com/cosmos/cosmos-sdk/blob/main/simapp/app.go#L200-L203
+```go
+processOpt := func(app *baseapp.BaseApp) {
+	app.SetProcessProposal(app.DefaultProcessProposal())
+}
+
+baseAppOptions = append(baseAppOptions, processOpt)
 ```
 
 ## Mempool
 
 Now that we have walked through the `PrepareProposal` & `ProcessProposal`, we can move on to walking through the mempool. 
 
-There are countless designs that an application developer can write for a mempool, the core team opted to provide a simple implementation of a nonce mempool. The nonce mempool is a mempool that keeps transactions from an sorted by nonce in order to avoid the issues with nonces. 
+There are countless designs that an application developer can write for a mempool, the SDK opted to provide only simple mempool implementations.
+Namely, the SDK provides the following mempools:
+
+* [No-op Mempool](#no-op-mempool)
+* [Sender Nonce Mempool](#sender-nonce-mempool)
+* [Priority Nonce Mempool](#priority-nonce-mempool)
 
+The default SDK is a [No-op Mempool](#no-op-mempool), but it can be replaced by the application developer in [`app.go`](./01-app-go-v2.md):
+
+```go
+nonceMempool := mempool.NewSenderNonceMempool()
+mempoolOpt   := baseapp.SetMempool(nonceMempool)
+baseAppOptions = append(baseAppOptions, mempoolOpt)
```

Comment:
There's no need to set and then append options.

Reply:
We do, `SetMempool` returns a `func(*BaseApp)`, which still need to be called.